src/ol/object.js

```diff
@@ -8,6 +9,33 @@ goog.require('ol.Observable');
 
 
 /**
+ * Polyfill for Object.assign().  Assigns enumerable and own properties from
+ * one or more source objects to a target object.
```

Comment:
Link added in https://github.com/tschaub/ol3/commit/34c705f150d8997aea9b976a76df7212ed2d4d54.